bioconda_utils/bioconda_utils-requirements.txt

```diff
@@ -21,3 +21,4 @@ colorlog=3.1.*
 six=1.11.*
 alabaster=0.7.*
 git=2.14.*
+conda-forge-pinning
```

Comment:
@johanneskoester should we pin this so that we can maintain control over when to move to the next set of variants?